### PR TITLE
add "decode" option to panel

### PR DESCRIPTION
**Pull request by qbalukom**

### CODE REVIEW

File: src/components/Extensibility/components/Panel.js [hunk 1]
```diff
@@ -44,6 +57,21 @@ export function Panel({
               />
             ))
           : null}
+        {decodable && (
+          <LayoutPanel.Actions>
+            <Button
+              option="transparent"
+              glyph={isDecoded ? 'hide' : 'show'}
+              //disabled={!secret?.data}
```
Comment (mrCherry97) [inline src/components/Extensibility/components/Panel.js:65]: probably it should be uncommented

**Review by mrCherry97 — CHANGES_REQUESTED** [review]
Code looks fine, I will test it a little bit and make re-review

File: src/components/Extensibility/components/Panel.js (hunk 1 above)
Comment (qbalukom) [reply src/components/Extensibility/components/Panel.js:65]: probably should be removed

**Review by mrCherry97 — DISMISSED** [review]

File: docs/extensibility/display-section.md [hunk 2]
```diff
@@ -524,6 +524,7 @@ Panel widgets render an object as a separate panel with its own title (based on
 - **header** - an optional array that allows you to, for example, display labels in the panel header.
 - **disablePadding** - an optional boolean which disables the padding inside the panel body.
 - **description** - displays a custom description on the resource list page. It can contain links. If the **translations** section has a translation entry with the ID that is the same as the **description** string, the translation is used.
+- **decodable** - decodes the values of all the children using base64 - has to be used together with the **source** parameter.
```
Comment (grego952) [inline docs/extensibility/display-section.md:527]:
```suggestion
- **decodable** - decodes the values of all the children using base64 - must be used together with the **source** parameter.
```

File: public/schemas/schema-details.yaml [hunk 3]
```diff
@@ -259,6 +259,9 @@ $widgets:
           type: boolean
         description:
           type: string
+        decodable:
+          type: boolean
+          description: Allow decoding of child values using base64
```
Comment (grego952) [inline public/schemas/schema-details.yaml:264]:
```suggestion
          description: Allows decoding of child values using base64
```

**Review by mrCherry97 — APPROVED** [review]

**Pull request merged**